app/views/layouts/_main_nav.html.erb

```diff
@@ -2,9 +2,6 @@
 <% other_nav_links = NavigationLink.other_section.ordered.to_a %>
 
 <nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-labelledby="default-nav-heading">
-  <h2 id="default-nav-heading" class="mb-1 fs-base">
-    <%= community_name %>
-  </h2>
```

Comment:
small change (blocking) - This heading was used on line 4 to label the navigation element... I agree we can bin the heading, but we'll need to add an aria-label to the nav instead e.g. `aria-label="<%= community_name %=>"` (I can't think of a better name, but maybe you can).

(not blocking) - TBH I'm not totally sure the "Other" heading makes much sense floating there either 🤷‍♀️   Maybe "Site info" would make more sense? Or maybe it's just me and it's fine how it is.

Reply:
1. I added the `aria-label`.
2. We're going to rework the navigation (again!) pretty soon, so I'd rather skip changing "Other" to avoid unnecessary confusion.